packages/autorest.typescript/test/rlcIntegration/urlRest.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { assert } from "chai";
 import UrlRest, {
+  getBooleanTrue,
```

Comment:
@joheredi  what if we have two operations with the same name from different operation group ?  How are customers going to make it work if they want to use both of them inside the same test file ?